shotover-proxy/Cargo.toml

```diff
@@ -33,6 +33,7 @@ pin-project-lite = "0.2"
 tokio-openssl = "0.6.2"
 openssl = { version = "0.10.36", features = ["vendored"] }
 async-recursion = "1.0"
+governor = "0.4.2"
```

Comment:
Can any of these default features be disabled? https://github.com/antifuchs/governor/blob/6eb1060686bb5e268ec27ea3a8c267338769a9ff/governor/Cargo.toml#L38

Reply:
It compiles for me with: `governor = { version = "0.4.2", default-features = false, features = ["std", "jitter"] }`
We will need to check if dashmap or quanta were needed for something important.